apps/pg_subscriptor/lib/handler.ex

```diff
@@ -96,6 +99,28 @@ defmodule PgHandler do
     <<_relation_oid::32, "N", _tuple_data::binary>> = body
   end
 
+  defp handle_update(body) do
+    Logger.info("Got UPDATE msg")
+    <<_relation_oid::32, rest::binary>> = body
+
+    {tuple_data, rest} =
```

Comment:
I know that you said that this is not ready yet, but I just wanted to mention that if I understand it correctly, there are 3 ways an `UPDATE` message could look like:
1. <<"U", 1::32, "N", "%{id: 1, name: Filip}"::binary>>
2. <<"U", 1::32, "K", "%{id: 1, name: Juraj}", "N", "%{id: 1, name: Filip}"::binary>>
3. <<"U", 1::32, "O", "%{id: 1, name: Juraj}", "N", "%{id: 1, name: Filip}"::binary>>
So I think handling of "N" should be outside of the `case`.